ParisBNF/et/BNFet343.xml

```diff
@@ -38,14 +38,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                     <listBibl type="catalogue">
                                         <bibl>
                                             <ptr target="bm:Grebaut1941GriauleII"/>
-                                            <citedRange unit="page">20-25</citedRange>                                           
+                                            <citedRange unit="page">58-61</citedRange>                                           
                                         </bibl>
                            
                                     </listBibl>
-                                 <listRelation>
```

Comment:
I do not understand these changes. 
We had a stub called Griaule 343. It has been renamed to BNF et 343, with a relation
If the description is for BNF et 350 then the filename should be changed and the relation updated accordingly to `<relation name="betmas:formerlyAlsoListedAs" active="BNFet350" passive="BNFgriaule343"/>`

If et 350 is a new record then BNFet343 should stay as is and BNFet350 created